e2e/system/build_from_git_test.go

```diff
@@ -12,10 +14,14 @@ func (s *Suite) TestBuildFromGit() {
 	s.T().Parallel()
 
 	// Setup
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	defer cancel()
 
+	s.T().Log("Creating new instance")
 	target, err := s.Knuu.NewInstance(namePrefix)
-	s.Require().NoError(err)
+	if err != nil {
+		s.T().Fatalf("Error creating new instance: %v", err)
+	}
```

Comment:
`s.Require().NoError(err)` does the same thing in one line. We can also set a custom message for it:

```go
s.Require().NoError(err, "some message")
```